articles/resources/relnotes.md

```diff
@@ -16,6 +16,21 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxx
+
+*Release date: July 1st, 2020*
+
+This release contains the following:
+
+- New qdk-chem chemistry format conversion tool 
+- New synthesis functionality in QuantumLibraries
+- IQ# now allows arguments to %simulate and %estimate
+- New phase display options in IQ#
+- Updated conda package support for IQ# installation to support local installation to a conda environment
+- When using the simulator, qubits no longer need to be in a 0 state upon release and instead can be measured or in a 0 state
+- Bug fixes 
```

Comment:
We should mention the new version roll-forward behavior here, especially since that has such an impact on the katas.

Reply:
any suggested language? What's the user impact to highlight here?

Reply:
The old behavior was that in order to use two Q# libraries together, they had to be built with the exact same version of the Quantum Development Kit. For the most part, that wasn't a huge problem given that the Quantum Development Kit isn't installed globally, but is referenced as an SDK from each csproj. When using IQ#, however, that meant that you needed the globally installed version of IQ# to match the package you wanted to use it with, something that was especially annoying for katas users running locally rather than online or in containers.

This change relaxes that so only the major and minor version have to match, making it much easier for IQ# users to make use of Q# library packages like the katas.

@rmshaffer's changes to allow implicit version numbers for the katas package and to allow conda environment–local rather than machine-local installations of IQ# also help this considerably, such that this release makes it much easier for users to consume libraries from IQ#.

+@tcNickolas for katas impact.